standard/documentation-comments.md

```diff
@@ -414,10 +434,15 @@ where
 **Example:**
 
 ```csharp
-/// <summary>Report a point's location as a string.</summary>
-/// <returns>A string representing a point's location, in the form (x,y),
-/// without any leading, trailing, or embedded whitespace.</returns>
-public override string ToString() {
+/// <summary>
+/// Report a point's location as a string.
+/// </summary>
+/// <returns>
+/// A string representing a point's location, in the form (x,y),
+/// without any leading, trailing, or embedded whitespace.
+/// </returns>
+public override string ToString()
+{
```

Comment:
One more space required for the return statement, I think.

Reply:
yes. I made this an expression bodied member instead.